packages/api/src/api/helix/moderation/HelixModerationApi.ts

```diff
@@ -55,6 +57,23 @@ export interface HelixCheckAutoModStatusData {
 
 export type HelixAutoModSettingsUpdate = Exclude<HelixAutoModSettings, 'broadcasterId' | 'moderatorId'>;
 
+export interface HelixBanUserRequest {
```

Comment:
All root-exported interfaces should have a description. (This one should also be exported at the root.)

Reply:
Fixed.